phpunit.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
+         bootstrap="./generis/test/bootstrap.php">
+    <testsuites>
+        <testsuite name="unit">
+            <directory>./*/test/unit</directory>
```

Comment:
Just a heads up that this is not the same path for all the repositories.

Reply:
That's why we have a bit more elaborate logic within `./test.sh`